libs/ngx-dsfr/search-bar/src/lib/search-bar.component.ts

```diff
@@ -131,12 +137,21 @@ export class DsfrSearchBarComponent implements ControlValueAccessor, OnDestroy,
   }
 
   ngAfterViewInit(): void {
-    if ( this.autocomplete ) {
+    this.writeValue( this._value );
+    this.initSearch();
+  }
+
+  private setSearchInput( content: ElementRef<HTMLInputElement> ): void {
+    if ( content ) {
+      this.searchInput = content;
       this.initSearch();
```

Comment:
It's more readable if you wrap the call within an `if` rather than what you did in lines 152-154